media_kit/lib/src/player/native/player/real.dart

```diff
@@ -2455,10 +2456,11 @@ class NativePlayer extends PlatformPlayer {
   }
 
   /// Adds an error to the [Player.stream.error].
-  void _error(int code) {
-    if (code < 0 && !errorController.isClosed) {
+  void _logError(int code, String? extraContext) {
+    if (code < 0 && !logController.isClosed) {
       final message = mpv.mpv_error_string(code).cast<Utf8>().toDartString();
-      errorController.add(message);
+      logController.add(PlayerLog(
+          prefix: "media_kit", level: 'error', text: 'error:$message $extraContext'));
```

Comment:
NIT: Trailing commas; single quote strings; adjust whitespaces.

```suggestion
          prefix: 'media_kit', level: 'error', text: 'error: $message text: $text'));
```